components/core/tests/test-ir_encoding_methods.cpp

```diff
@@ -273,28 +259,38 @@ TEMPLATE_TEST_CASE("decode_preamble", "[ffi][decode_preamble]", four_byte_encode
 
     // Test if preamble can be decoded correctly
     TimestampInfo ts_info;
-    epoch_time_ms_t decoded_ts;
-    REQUIRE(decode_preamble<TestType>(preamble_buffer, ts_info, decoded_ts) ==
+    int8_t metadata_type;
+    size_t metadata_pos;
```

Comment:
Is this variable used anywhere in the test?

Reply:
There is a check for the correct type on 274, so it is used.

I didn't use pos in the test, since the afaik the only way to check it would be to hardcode the number of bytes in the preamble up to that point. Any better ideas?

Either one can't be removed since they are return values essentially (passed by reference).

I'm pushing a commit to initialize them to zero, since I wanted to, but forgot.